crates/pixi_manifest/src/task.rs

```diff
@@ -48,26 +48,45 @@ impl From<String> for TaskName {
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize)]
 pub struct Dependency {
     pub task_name: TaskName,
-    pub args: Option<Vec<String>>,
+    pub args: Option<Vec<TemplateString>>,
     pub environment: Option<EnvironmentName>,
 }
 
 impl Dependency {
-    pub fn new(s: &str, args: Option<Vec<String>>, environment: Option<EnvironmentName>) -> Self {
+    pub fn new(
+        s: &str,
```

Comment:
Instead of `s: &str` could this be `s: TemplateString`? so that the `{ task = "{{ a_task_name }}", ... }` could be supported?